src/sdgio-edit.owl

```diff
@@ -3293,6 +3307,45 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020299> "increased capacity for innovation in partner development organizations and in poor and vulnerable communities"@en)
```

Comment:
Same issue on phrasing, look across all objectives and correct.

Reply:
yes, I posted a proposal in the google spreadsheet. However, if I understand, you are asking to correct right away. So, I will do it now.